src/sentry/snuba/metrics/extraction.py

```diff
@@ -40,7 +40,9 @@
 
 SPEC_VERSION_TWO_FLAG = "organizations:on-demand-metrics-query-spec-version-two"
 # Certain functions will only be supported with certain feature flags
-OPS_REQUIRE_FEAT_FLAG: dict[str, str] = {}
+OPS_REQUIRE_FEAT_FLAG: dict[str, str] = {
+    "user_misery": SPEC_VERSION_TWO_FLAG,
```

Comment:
This means that the only way to query for this function is if an organization has the feature flag enabled.